mobx/lib/src/core/observable.dart

```diff
@@ -1,11 +1,28 @@
 part of '../core.dart';
 
-class ObservableValue<T> extends Atom
+class Observable<T> extends Atom
     implements Interceptable<T>, Listenable<ChangeNotification<T>> {
-  ObservableValue(ReactiveContext context, this._value, {String name})
+  /// Create an observable value with an [initialValue] and an optional [name]
+  ///
+  /// Observable values are tracked inside MobX. When a reaction uses them
+  /// they are implicitly added as a dependency of the reaction. When its value changes
+  /// the linked reaction is re-triggered.
+  ///
+  /// An Observable's value is read with the `value` property.
+  ///
+  /// ```
+  /// var x = Observable(10);
+  /// var message = Observable('hello');
+  ///
+  /// print('x = ${x.value}'); // read an Observable's value
+  /// ```
+  factory Observable(T initialValue, {String name, ReactiveContext context}) =>
+      Observable._(context ?? mainContext, initialValue, name: name);
+
+  Observable._(ReactiveContext context, this._value, {String name})
       : _interceptors = Interceptors(context),
```

Comment:
May be we can init the **interceptors** and **listeners** lazily, just like you did for `ObservableList<T>` ?

Reply:
Let's do it in another PR

Reply:
agree..